synapse/lib/httpapi.py

```diff
@@ -1418,17 +1411,18 @@ async def _runHttpExt(self, meth, path):
 
         except Exception as e:
             rcode = True
-            enfo = s_common.err(e)
-            logger.exception(f'Extended HTTP API {iden} encountered fatal error: {enfo[1].get("mesg")}')
+            errname, errinfo = s_common.err(e)
+            extra = core.getLogExtra(httpapi=iden, errname=errname, **errinfo)
+            logger.exception(f'Extended HTTP API encountered a fatal error.', extra=extra)
             if rbody is False:
                 self.clear()
                 self.set_status(500)
-                self.sendRestErr(enfo[0],
-                                 f'Extended HTTP API {iden} encountered fatal error: {enfo[1].get("mesg")}')
+                self.sendRestErr(errname,
+                                 f'Extended HTTP API {iden} encountered fatal error: {errinfo.get("mesg")}')
 
         if rcode is False:
             self.clear()
             self.set_status(500)
-            self.sendRestErr('StormRuntimeError', f'Extended HTTP API {iden} never set status code.')
+            self.sendRestErr('StormRuntimeError', 'Extended HTTP API never set status code.')
```

Comment:
Why is the iden being removed here?

Reply:
It seemed like an unnecessary disclosure of "internals" to a caller who knows what API endpoint they hit. Happy to discuss. :+1: